### DISCUSSION TITLE
Qwik integration

### DISCUSSION THREAD

**Discussion by gioboa** [trpc/trpc#2386] [Ideas]
Hi 👋 I would love to create tRPC Qwik library ❤️
I will working on it 💪

**Comment by sachinraja** [comment]
Hey, this should definitely be possible using `@trpc/client` right now. Not sure if Qwik offers primitives for a closer integration, but that would great, feel free to ask any questions you have here or on the Discord.

**Comment by gioboa** [comment]
I created tRPC Qwik City adapter
library --> https://github.com/gioboa/trpc-qwik-city
example with prisma --> https://github.com/gioboa/trpc-qwik-city-example
I took inspiration from trpc-sveltekit adapter thanks @icflorescu

**Reply by sachinraja** [reply]
Awesome! You can also check out our [official fetch adapter](https://alpha.trpc.io/docs/fetch) for v10, which should work with SvelteKit, Qwik, etc.
